src/components/Section/Citizenship/Citizenship.jsx

```diff
@@ -1,198 +1,98 @@
+        {para}
 
-          <SectionView
-            name="status"
-            back="citizenship/intro"
-            backLabel={i18n.t('citizenship.destination.intro')}
-            next="citizenship/multiple"
-            nextLabel={i18n.t('citizenship.destination.multiple')}>
-            <Status
-              name="status"
-              {...this.props.Status}
-              dispatch={this.props.dispatch}
-              onUpdate={this.handleUpdate.bind(this, 'Status')}
-              onError={this.handleError}
-            />
-          </SectionView>
+        <div className={subsectionClasses}>
+          {isReview && (
+            <div className="top-btns"><ErrorList /></div>
+          )}
 
-          <SectionView
-            name="multiple"
-            back="citizenship/status"
-            backLabel={i18n.t('citizenship.destination.status')}
-            next="citizenship/passports"
-            nextLabel={i18n.t('citizenship.destination.passports')}>
-            <Multiple
-              name="multiple"
-              {...this.props.Multiple}
-              dispatch={this.props.dispatch}
-              onUpdate={this.handleUpdate.bind(this, 'Multiple')}
-              onError={this.handleError}
-              scrollToBottom={this.props.scrollToBottom}
-            />
-          </SectionView>
+          {this.getCitizenshipSubsections()}
 
-          <SectionView
-            name="passports"
-            back="citizenship/multiple"
-            backLabel={i18n.t('citizenship.destination.multiple')}
-            next="citizenship/review"
-            nextLabel={i18n.t('citizenship.destination.review')}>
-            <Passports
-              name="passports"
-              {...this.props.Passports}
-              dispatch={this.props.dispatch}
-              onUpdate={this.handleUpdate.bind(this, 'Passports')}
-              onError={this.handleError}
-              scrollToBottom={this.props.scrollToBottom}
-            />
-          </SectionView>
-        </SectionViews>
+          <SectionNavigation
+            section={CITIZENSHIP}
+            subsection={subsection}
+            formType={formType}
+          />
+        </div>
       </div>
     )
   }
 }
 
 function mapStateToProps(state) {
-  let app = state.application || {}
-  let citizenship = app.Citizenship || {}
-  let errors = app.Errors || {}
-  let completed = app.Completed || {}
+  const app = state.application || {}
+  const auth = state.authentication || {}
+  const citizenship = app.Citizenship || {}
+  const errors = app.Errors || {}
+  const completed = app.Completed || {}
+
   return {
     Application: app,
     Citizenship: citizenship,
     Status: citizenship.Status || {},
     Multiple: citizenship.Multiple || {},
     Passports: citizenship.Passports || {},
     Errors: errors.citizenship || [],
-    Completed: completed.citizenship || []
+    Completed: completed.citizenship || [],
+    formType: auth.formType
```

Comment:
This should also return `...state.section` so `Citizenship` receives the current subsection (see https://github.com/18F/e-QIP-prototype/blob/develop/src/components/Section/Identification/Identification.jsx#L73). Also, since the only props Citizenship uses are `subsection, formType`, the others can be removed.